code-reviews.md

```diff
@@ -0,0 +1,141 @@
+# Code Review Guide
+
+> A friendly guide for reviewing code--and not each other--at CFPB.
+
+## Why reviews?
+
+Code reviews are incredibly important step to the development process. Our goals
+when reviewing code are to:
+
+- expose bugs before they make it to production
+- ensure a concistent quality in the codebase
```

Comment:
`concistent` > `consistent`

While you're in there, FWIW if all bullets are sentences some style guides would say they should end with periods (and be capitalized).